src/translate_legacy.ts

```diff
@@ -400,7 +400,7 @@ export function translateHomeValues(flat:any): HEScoreJSONSchema {
                 !keys.some(
                     (key) =>
                         !isEmpty(clearedObj[key]) &&
-                        !['side', 'name', 'hvac_name', 'roof_name', 'floor_name'].includes(key) &&
```

Comment:
You explain why this was causing the bug?

Reply:
For an assessment represented above we are current generate JSON like 
```
 "zone_wall": [
            {
                "side": "front",
                "adjacent_to": "outside",
                "wall_assembly_code": "ewsb00st",
                "zone_window": {
                    "window_area": 22,
                    "window_method": "custom",
                    "window_u_value": 0.44,
                    "window_shgc": 0.55,
                    "solar_screen": false
                }
            },
            {
                "side": "back",
                "adjacent_to": "other_unit"
            }
        ]
```

and the JSON schema validation is failed as follows and will show the UI above

```
Value "front" does not match const.
Value "back" does not match const.
```

With the change, the JSON is

 ```
 "zone_wall": [
            {
                "side": "front",
                "adjacent_to": "outside",
                "wall_assembly_code": "ewsb00st",
                "zone_window": {
                    "window_area": 22,
                    "window_method": "custom",
                    "window_u_value": 0.44,
                    "window_shgc": 0.55,
                    "solar_screen": false
                }
            },
            {
                "side": "back",
                "adjacent_to": "other_unit"
            },
            {
                "side": "left",
            },
            {
                "side": "right",
            }
        ]
```

and the error is 

`Required properties are missing from object: adjacent_to.`